From 6dd78e86b3d242aeadbf74a1279c31c95727db16 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Thu, 12 Jun 2025 22:11:24 -0400
Subject: [PATCH] Fix ex_doc deprecation warning

---
 mix.exs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mix.exs b/mix.exs
index 4e0e796..277513f 100644
--- a/mix.exs
+++ b/mix.exs
@@ -77,7 +77,7 @@ defmodule VintageNetWizard.MixProject do
 
   defp docs do
     [
-      assets: "assets",
+      assets: %{"assets" => "assets"},
       extras: ["README.md", "json-api.md", "CHANGELOG.md"],
       main: "readme",
       source_ref: "v#{@version}",
-- 
2.52.0

