From 000ecf0d42b222256d3c8661ac0b1c0f4f82cbae Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Thu, 22 Jan 2026 21:49:00 -0500
Subject: [PATCH] Add assets to docs

---
 mix.exs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mix.exs b/mix.exs
index bca6ea2..30d163e 100644
--- a/mix.exs
+++ b/mix.exs
@@ -69,6 +69,7 @@ defmodule VintageNetEthernet.MixProject do
 
   defp docs do
     [
+      assets: %{"assets" => "assets"},
       extras: ["README.md", "CHANGELOG.md"],
       main: "readme",
       source_ref: "v#{@version}",
-- 
2.52.0

